src/material.scss

```diff
@@ -20,9 +20,18 @@
 // Include the common styles for Angular Material. We include this here so that you only
 // have to load a single css file for Angular Material in your app.
 // Be sure that you only ever include this mixin once!
-@include mat.core();
+// As of v15 mat.legacy-core no longer includes default typography styles.
```

Comment:
This block created by migration tool. I'm not super familiar with typography, but the pages look the same as before, so leaving it as is.